src/main/java/org/humancellatlas/ingest/core/service/MetadataLinkingService.java

```diff
@@ -14,6 +16,9 @@
 @Service
 public class MetadataLinkingService {
 
+    private static final long RETRY_BACKOFF_MS = 100;
```

Comment:
Are you sure 100 ms is the correct value for retry (backoff)? Have you tested this with other values? Tx

Reply:
I was able to reproduce the issue in the ebi-ait/hca-ebi-wrangler-central#600  locally, and applying this retry backoff works fine. We could adjust accordingly if we find it's not enough.